capirca/lib/paloaltofw.py

```diff
@@ -884,6 +882,28 @@ def __str__(self):
           member = etree.SubElement(from_, "member")
           member.text = x
 
+        dest = etree.SubElement(entry, "destination")
```

Comment:
We now have destination always rendered prior to the source in a rule (which to me is slightly counterintuitive). 

Is this how the device displays it if I display the xml config? While this doesn't have to be fixed in this commit, I am thinking about the long term adherence to this [requirement](https://github.com/google/capirca/blob/master/doc/generator_patterns.md#syntax-of-the-config-from-the-generator-and-on-device-should-match-when-cryptographically-verified).